packages/plugin-core/src/commands/ShowPreviewV2.ts

```diff
@@ -139,28 +133,30 @@ export class ShowPreviewV2Command extends BasicCommand<
   }
 
   async execute(_opts?: CommandOpts) {
-    // Get workspace information
+    const ctx = "ShowPreview";
     const ext = getExtension();
-
-    // If panel already exists
     const existingPanel = ext.getWebView(DendronWebViewKey.NOTE_PREVIEW);
-
     const viewColumn = vscode.ViewColumn.Beside; // Editor column to show the new webview panel in.
     const preserveFocus = true;
 
+    // show existing panel if exist
     if (!_.isUndefined(existingPanel)) {
+      Logger.info({ ctx, msg: "reveal existing" });
       try {
         // If error, panel disposed and needs to be recreated
         existingPanel.reveal(viewColumn, preserveFocus);
         return;
-      } catch (error) {
-        console.error(error);
+      } catch (error: any) {
+        Logger.error({ ctx, error });
       }
     }
+    Logger.info({ ctx, msg: "creating new" });
 
+    const root =
+      "/Users/kevinlin/code/dendron/packages/dendron-plugin-views/build";
```

Comment:
Hardcoded to your own user path.

Reply:
yeah, this was draft - fixed in other pr